lib/transactions.js

```diff
@@ -288,9 +288,48 @@ exports.createGeneration = (rpcData, blockReward, feeReward, recipients, poolAdd
                 Math.round(masternodeReward)
             )
         }
+    } else {
+        // case for ZelCash
+        let zelnodeBasicAddrHash = zelnodeBasicAddress ? bitcoin.address.fromBase58Check(zelnodeBasicAddress).hash : null
+        let zelnodeSuperAddrHash = zelnodeSuperAddress ? bitcoin.address.fromBase58Check(zelnodeSuperAddress).hash : null
+        let zelnodeBamfAddrHash = zelnodeBamfAddress ? bitcoin.address.fromBase58Check(zelnodeBamfAddress).hash : null
+        zelnodeBasicAmount = zelnodeBasicAmount || 0;
+        zelnodeSuperAmount = zelnodeSuperAmount || 0;
+        zelnodeBamfAmount = zelnodeBamfAmount || 0;
```

Comment:
Can you move the amount `|| 0` to `blockTemplate.js:71-75`?

Reply:
implemented with latest commit